src/components/pages/SavedCases/SavedCases.js

```diff
@@ -39,6 +41,28 @@ function SavedCases({ savedCases, setSavedCases, deleteBookmark }) {
       ),
       width: '5%',
     },
+    {
+      title: 'Download Report',
+      render: (text, record) => (
+        <PDFExportButton
+          key="export"
+          fileName={`Judge_${record.last_name}_Case_${record.case_number}`}
```

Comment:
Excellent file name convention

Reply:
Thanks.
I wanted to make sure that the filename was both unique and easily to determine who/what.